README.md

```diff
@@ -577,6 +579,70 @@ $data = [
 
 > [1, 2, 3, 3, 4, 5]
 
+### IfElse
+
+Delegates to one expression or another depending on whether the specified condition strictly evaluates to true.
+
+If the condition does not return a boolean an InvalidReturnException will be thrown.
+
+#### Signature
+
+```php
+IfElse(callable $condition, Strategy|Mapping|array|mixed $if, Strategy|Mapping|array|mixed $else = null)
+```
+
+ 1. `$condition` &ndash; Condition.
+ 2. `$if` &ndash; Expression used when condition loosely evaluates to true.
+ 3. `$else` &ndash; Expression used when condition loosely evaluates to false.
+
+#### Example
+
+Test if all items in $data are positive.
+
+```php
+$data = [1, 3, 5, 6];
```

Comment:
Although I regret it is not documented how to write documentation(!), in general we aim to keep examples as simple as possible. Although we use sets to demonstrate some strategies, such as `Filter`, to demonstrate filtering a set, it seems we do not need to use a set just to demonstrate functionality of `IfElse`. Consider just using simple (in)equality tests instead to simplify this example.

Our second objective when writing documentation is to keep it as short as possible whilst still explaining everything fully. We probably don't need to demonstrate both the *if* and *else* cases since most developers probably understand how if/else works already! One example should suffice, as with [either](https://github.com/ScriptFUSION/Mapper#either).